xbmc/windowing/gbm/drm/DRMAtomic.cpp

```diff
@@ -267,6 +249,32 @@ bool CDRMAtomic::AddProperty(CDRMObject* object, const char* name, uint64_t valu
   return true;
 }
 
+bool CDRMAtomic::AddPropertyValueByName(struct drm_object* object,
+                                        const char* name,
+                                        const char* type)
+{
+  for (uint32_t i = 0; i < object->props->count_props; i++)
+  {
+    if (!strcmp(object->props_info[i]->name, name))
```

Comment:
This might be easier to read if you used `continue` when the name does not match. The same for the other `strcmp()` below.